dolphins.json

```
@@ -0,0 +1,18 @@
+[
+  "Giraffes only sleep 5 minutes a day.",
+  "Dolphins once lived on land, but decided it was a terrible idea and went back to the sea",
+  "Dolphins once looked and behaved a lot like wolves, only with small, giraffe-like hoof-toes on each foot instead of claws.",
+  "Dolphins once had fingers. They still have remnants of finger bones in their flippers, along with a forearm, wrists, and few remnants of leg bones deep inside their bodies.",
+  "Dolphins always remember to :breathe:. They replace 90% of the air in their lungs every single time they breathe in. Humans only replace 15% of the air in their lungs when they breathe.",
+  "Some dolphins can leap 20 feet in the air. If they're showing off, they'll do backflips, too.",
+  "Dolphins don’t dream.",
+  "Dolphins are incredibly light sleepers. You can wake them up just by walking near them. But that would be hard, because of the ocean.",
+  "Unlike vampires, dolphins can recognize themselves in the mirror.",
+  "In Ancient Greece, killing a dolphin was considered sacrilegious and was punishable by death. Our standards today are not as strict, much to the chagrin of the dolphins.",
+  "Five species of dolphins live in rivers. They like to go tubing.",
+  "Dolphins have the longest memory in the animal kingdom. Elephants often forget this.",
+  "Dolphins are able to talk and understand each other over the phone, but they have no pocket for their phone, which makes it harder.",
+  "Dolphin’s exhale from their blowholes at speeds of 100 miles per hour, a feat only matched by Redditors.",
+  "Dolphins have belly buttons.",
+  "Blocking hear mostly through their lower jaw. If you block off a dolphin’s ears it will hardly have any effect on its hearing."
```

Should this be:
```suggestion
  "Dolphins hear mostly through their lower jaw. If you block off a dolphin’s ears it will hardly have any effect on its hearing."
```
?